### PR TITLE
[Task.45600] Làm payment

### PR DESCRIPTION
**Pull request by ghost**
https://edu-redmine.sun-asterisk.vn/issues/45600
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/Component/Cart/index.jsx [hunk 1]
```diff
@@ -3,19 +3,30 @@ import { Container } from "react-bootstrap";
...
+  let total = useSelector((state) => state.cart.total)
   let newTotal = total.toLocaleString('it-IT')
   const [cartListRender, setCartListRender] = useState();
 
+  const userInfo = JSON.parse(localStorage.getItem('user-info'));
+  const userPaymentInfoLocal = JSON.parse(localStorage.getItem('user-payment-info'));
+  const userPaymentInfo = useSelector((state) => state.cart.payment_info)
+
+  const usePaymentCheck = userPaymentInfoLocal ? userPaymentInfoLocal : userPaymentInfo;
+  const checkForConfirm = Object.values(userPaymentInfo).some(item => item == "");
```
Comment (vietnq-1097) [inline src/Component/Cart/index.jsx:26]:
```suggestion
  const checkForConfirm = Object.values(userPaymentInfo).some(Boolean);
```
Thử xem nhé

File: src/Component/Cart/index.jsx [hunk 2]
```diff
@@ -3,19 +3,30 @@ import { Container } from "react-bootstrap";
...
+  console.log(userPaymentInfo)
```
Comment (vietnq-1097) [inline src/Component/Cart/index.jsx]: Xóa console

**Comment by ghost** [comment]
ready

**Pull request merged**